paddle/fluid/platform/device_tracer.cc

```diff
@@ -130,8 +130,6 @@ void CUPTIAPI bufferRequested(uint8_t **buffer, size_t *size,
   uint8_t *buf = reinterpret_cast<uint8_t *>(malloc(kBufSize + kAlignSize));
   *size = kBufSize;
   *buffer = ALIGN_BUFFER(buf, kAlignSize);
-  free(buf);
```

Comment:
I didn't notice that it was requesting a buffer.... and I found that has been freed through the cuptiActivityRegisterCallbacks ， and finally freed at the end of bufferCompleted function.